man/man/man1m/mkzpool.1m

```diff
@@ -0,0 +1,41 @@
+.Dd November 13, 2019
```

Comment:
Date needs to be updated

Reply:
The date on man pages is usually the date of last modification (that's how it is for all of the illumos man pages).

Reply:
Discussed with @jasonbking, this is accurate as to when the change was authored, so I call it good.